zipkin-cassandra/src/schema/cassandra-schema-cql3.txt

```diff
@@ -0,0 +1,73 @@
+CREATE KEYSPACE "Zipkin" WITH replication = {'class': 'NetworkTopologyStrategy', 'us-west': '1'}  AND durable_writes = true;
```

Comment:
You probably want 'us-west': '3' otherwise you are only getting one copy of your data.